.circleci/config.yml

```diff
@@ -8,105 +11,117 @@ references:
+      - run: yarn test-ci
       - save_cache:
           key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
           paths:
             - /usr/local/share/.cache/yarn/v2
             - ./node_modules
 
-  test:
+  build_linux:
     <<: *container_config
     steps:
       - checkout
       - restore_cache:
           key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
       - run: apt-get -y update
-      - run: apt-get -y install libusb-1.0-0-dev graphicsmagick libudev-dev
-      - run: apt-get -y install libxtst6 libxss1 libgtk2.0-0 libnss3 libasound2 libgconf-2-4
-      - run: yarn test-ci
-    store_artifacts:
-      - path: artifacts/
-      - destination: yarnpkg
+      - run: apt-get -y install libusb-1.0-0-dev icnsutils graphicsmagick libudev-dev
+      - run: yarn config delete proxy
+      - run: yarn install --ignore-engines --frozen-lockfile --network-timeout 1000000 --network-concurrency 1
+      - run: yarn dist
+      - store_artifacts:
+          path: dist
+          destination: build
 
-  deploy_win64:
-    <<: *win_config
+  build_macos:
+    macos:
+      xcode: "12.0.0"
     steps:
       - checkout
       - restore_cache:
           key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
-      - run: apt-get -y update
-      - run: apt-get -y install libusb-1.0-0-dev icnsutils graphicsmagick libudev-dev
-      - run: apt-get install --no-install-recommends -y gcc-multilib g++-multilib
       - run: yarn install --ignore-engines --frozen-lockfile --network-timeout 1000000 --network-concurrency 1
-      - run: yarn assets
-      - run: yarn electron-builder -w --x64
+      - save_cache:
+          key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
+          paths:
+            - /usr/local/share/.cache/yarn/v2
+            - ./node_modules
+      - run: yarn dist
       - store_artifacts:
           path: dist
           destination: build
 
-  deploy_linux:
-    <<: *container_config
+
+  build_win:
+    executor:
+      name: win/default
+      size: "medium"
     steps:
       - checkout
-      - restore_cache:
-          key: neon-wallet-dependencies-cache-{{ checksum "yarn.lock" }}
-      - run: apt-get -y update
-      - run: apt-get -y install libusb-1.0-0-dev icnsutils graphicsmagick libudev-dev
-      - run: yarn config delete proxy
+      - run: npm install --global --production windows-build-tools --vs2015
+      - run: npm config set msvs_version 2015 -g
+      - run: choco install windows-sdk-8.1
+      # TODO: implenent caching
+      # - restore_cache:
+      #     key: neon-wallet-dependencies-windows-cache-{{ checksum "yarn.lock" }}
       - run: yarn install --ignore-engines --frozen-lockfile --network-timeout 1000000 --network-concurrency 1
-      - run: yarn lint
+      # - save_cache:
+      #     key: neon-wallet-dependencies-windows-cache-{{ checksum "yarn.lock" }}
+      #     paths:
+      #       - /usr/local/share/.cache/yarn/v2
```

Comment:
I couldn't figure out what the path was to cached yarn modules on windows 🧐 will circle back to this in the future